core/server/data/meta/asset_url.js

```diff
@@ -39,11 +39,13 @@ function getAssetUrl(path, isAdmin, minify) {
         // we don't want to concat the path with our favicon url
         output += path;
 
-        if (!config.get('assetHash')) {
-            config.set('assetHash', utils.generateAssetHash());
-        }
+        if (config.get('env') === 'production') {
```

Comment:
This might actually be unnecessary, the asset hash wasn't what was causing the issue. However @JohnONolan  mentioned that we shouldn't be using asset hashes in development, so this removed them.